Example/FinancialConnections Example/FinancialConnectionsUITests/InstantDebitsUITests.swift

```diff
@@ -108,4 +108,32 @@ final class InstantDebitsUITests: XCTestCase {
         app.fc_nativeConnectAccountsButton.tap()
         app.fc_nativeSuccessDoneButton.tap()
     }
+    
+    func test_connect() {
+        let app = XCUIApplication.fc_launch(playgroundConfigurationString:
+            """
+            {"use_case":"payment_intent","experience":"instant_debits","sdk_type":"native","test_mode":true,"merchant":"connect","payment_method_permission":true}
+            """
+        )
+
+        app.fc_playgroundCell.tap()
+        app.fc_playgroundShowAuthFlowButton.tap()
+
+        app.fc_nativeConsentAgreeButton.tap()
+
+        let emailTextField = app.textFields["email_text_field"]
+        XCTAssertTrue(emailTextField.waitForExistence(timeout: 10.0), "Failed to find email text field")
+        emailTextField.typeText("email@email.com")
```

Comment:
I think we could also pass the email in the playground configuration and skip this typing, but this is also good!